src/components/navmenu.js

```diff
@@ -37,21 +37,13 @@ export default function () {
         currentClassName="current"
         offset={-100}
       >
-        {menuItems.map((value, index) => {
-          return (
-            <li key={String(index)}>
-              <button
-                type="button"
-                onClick={() => {
-                  scrollTo(`#${value.path}`);
-                  setVisibility(0);
-                }}
-              >
-                {value.label}
-              </button>
-            </li>
-          );
-        })}
+        <li key="0">
+          <button type="button">
+            <Link to="/get-started" className="NavLink">
+              GET STARTED
```

Comment:
I think you don't need to write this in caps.

Reply:
So all letters without caps, or only the "G" and the "S" ?

Reply:
Get Started

Reply:
Are you sure about that? I don't know why I feel the full caps version is more appealing to my eyes at least.  😆 

![Screen Shot 2020-09-28 at 8 57 58 PM](https://user-images.githubusercontent.com/19671041/94468577-53283700-01cd-11eb-96b6-79eecaff669f.png)